docs/pom.xml

```diff
@@ -38,4 +38,61 @@
             </plugin>
         </plugins>
     </build>
+    <profiles>
+        <profile>
+            <id>release</id>
```

Comment:
It overrides the build-helper plugin because each app has a slightly different set of docs. Some don't have Extending, some have Using, etc. 
The build-helper plugin doesn't support wildcards, so each app has to explicitly list the files to include. Bringing in the profile here keeps the build from breaking when the files aren't generated.